IKEA/IKEA/TabBarController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  TabBarController.swift
+//  IKEA
+//
+//  Created by Yu ahyeon on 2022/10/02.
+//
+
+import UIKit
+
+class TabBarController: UITabBarController {
+    
+    // MARK: - property
+    
+    private lazy var mainViewController: UIViewController = {
+        $0.view.backgroundColor = .red
+        $0.tabBarItem = UITabBarItem(
+            title: nil,
+            image: UIImage(systemName: "house"),
+            tag: 1
+        )
+        return $0
+    }(UIViewController())
+    
+    private lazy var searchViewController: UIViewController = {
+        $0.tabBarItem = UITabBarItem(
+            title: nil,
+            image: UIImage(systemName: "text.magnifyingglass"),
+            tag: 2
+        )
+        return $0
+    }(UIViewController())
+    
+    private lazy var userInfoViewController: UIViewController = {
+        $0.view.backgroundColor = .yellow
+        $0.tabBarItem = UITabBarItem(
+            title: nil,
+            image: UIImage(systemName: "person"),
+            tag: 3
+        )
+        return $0
+    }(UIViewController())
+    
+    private lazy var wishlistViewController: UIViewController = {
+        $0.view.backgroundColor = .green
+        $0.tabBarItem = UITabBarItem(
+            title: nil,
+            image: UIImage(systemName: "heart"),
+            tag: 4
+        )
+        return $0
+    }(UIViewController())
+    
+    private lazy var cartViewController: UIViewController = {
+        $0.view.backgroundColor = .blue
+        $0.tabBarItem = UITabBarItem(
+            title: nil,
+            image: UIImage(systemName: "cart"),
+            tag: 5
+        )
+        return $0
+    }(UIViewController())
+    
+    // MARK: - life cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupAttributes()
+    }
+    
+    // MARK: - func
+    
+    func setupAttributes() {
+        view.backgroundColor = UIColor.systemBackground
+        setupTabBar()
+    }
+    
+    func setupTabBar() {
+        self.viewControllers =
+        [mainViewController,
+         searchViewController,
+         userInfoViewController,
+         wishlistViewController,
+         cartViewController
+        ]
```

Comment:
p4;
인덴테이션 방식에 조금 변화를 주면 어떨까요?

```suggestion
        self.viewControllers = [
            mainViewController,
            searchViewController,
            userInfoViewController,
            wishlistViewController,
            cartViewController
        ]
```

Reply:
와 뭔가 불편하다 했는데, 스누피 피드백에 편-안해 지네요..

Reply:
앗 그러네요! 수정했습니다!
감사합니다 스누피 🥰

e8ab89f9a899b5af9932f3e4e9f61285e074baaa

Reply:
이런 방식도 괜찮을 것 같네요 ㅎ
```Swift
        self.viewControllers = [mainViewController,
                                searchViewController,
                                userInfoViewController,
                                wishlistViewController,
                                cartViewController]
```

Reply:
> 이런 방식도 괜찮을 것 같네요 ㅎ
> 
> ```swift
>         self.viewControllers = [mainViewController,
>                                 searchViewController,
>                                 userInfoViewController,
>                                 wishlistViewController,
>                                 cartViewController]
> ```

오 이런 방법도 있었군요! 가독성 면에서 이 방식도 좋은 것 같아요!
또 하나 배워 갑니다! 감사합니다!!☺️